source/util/hex_float.h

```diff
@@ -1072,7 +1096,7 @@ std::istream& operator>>(std::istream& is, HexFloat<T, Traits>& value) {
       HF::exponent_mask);
   output_bits |= shifted_exponent;
 
-  T output_float = spvutils::BitwiseCast<T>(output_bits);
+  T output_float = T(output_bits);
```

Comment:
May as well use constructor form and skip the assignment.

T output_float(output_bits);